docs/source/whatsnew/pr/interactive_assignment.rst

```diff
@@ -0,0 +1,18 @@
+IPython can now trigger the display hook on last assignment of cells. 
+Up until 6.0 the following code wouldn't show the value of the assigned
+variable:
+
+    >>> var = "something"
+    # nothing shown
+
+You would have to actually make it the last statement:
+
+    >>> var = "someting else"
+    ... var 
+    "something else"
+
+With the option ``InteractiveShell.ast_node_interactivity='last_expr_or_assign'``
+you can now do::
+
+    >>> df = pandas.load_csv(...)
```

Comment:
Make this example match the others above? And use IPython prompts instead of `>>>`?